Makefile

```diff
@@ -262,9 +275,9 @@ audit: sign audit-licenses
 
 publish: audit publish-bin publish-pip publish-src publish-jars
 
-all: clean test audit
+all: clean system-test test audit
```

Comment:
Not sure if it matters, but if order of the dependencies is the same as order of run in make, then we should do `test` before `system-test`.